python_modules/libraries/dagster-duckdb-pandas/dagster_duckdb_pandas/duckdb_pandas_type_handler.py

```diff
@@ -66,3 +73,69 @@ def load_input(self, context: InputContext, table_slice: TableSlice) -> pd.DataF
     @property
     def supported_types(self):
         return [pd.DataFrame]
+
+
+# Helper function used as a decorator below
+# The only purpose in doing this is so that
+# we have a symbol to hang the snowflake_pandas_io_manager
+# docblock off of. Otherwise we would just invoke build_snowflake_io_manager
+# directly and assign to module-scoped variable
+def wrap_io_manager(fn) -> IOManagerDefinition:
```

Comment:
would moving this to the db_io_manager base class file make sense?

Reply:
I think this one of those cases where the function is so trivial that making it shared just causes more confusion